jdxsmith/whats-cookin
# Feature/html css styling

#### What does this PR do?
Styles the page including:
- Randomized banner with "Recipe of the Day"
- Randomized user button that generates a new user that updates the pantry
- Makes the active state for category buttons noticeable using a filter
- Updates favorites and recipes to cook icons
- Styles recipe cards
- Adds banner to pantry page
- Adds randomized colored rectangle to pantry cards

#### Where should the reviewer start?
- `scripts.js`, `styles.css` and `index.html`

#### How should this be manually tested?
- Run `index.html` in the browser. Click around to ensure that formatting still upkeeps functionality.

#### Any background context you want to provide?
- There are some small formatting changes that will need to be revisited including the commas on the recipe card

#### Screenshots (if appropriate)
Home Page Banner
<img width="1344" alt="Screen Shot 2020-10-18 at 5 53 02 PM" src="https://user-images.githubusercontent.com/58453566/96389134-e4565200-116a-11eb-893e-2c150ff03020.png">

Active State for Category Buttons
<img width="412" alt="Screen Shot 2020-10-18 at 5 53 12 PM" src="https://user-images.githubusercontent.com/58453566/96389147-f932e580-116a-11eb-807a-9a63519618de.png">

Updated Recipe Card Buttons
<img width="243" alt="Screen Shot 2020-10-18 at 5 53 19 PM" src="https://user-images.githubusercontent.com/58453566/96389164-1071d300-116b-11eb-8597-ffe9a201e266.png">

Updated Pantry View
<img width="1338" alt="Screen Shot 2020-10-18 at 5 53 31 PM" src="https://user-images.githubusercontent.com/58453566/96389171-17004a80-116b-11eb-8b0d-f7d503b5698d.png">

#### Questions:
- Any styling that you don't like or want changed?
